src/main/java/fi/otavanopisto/kuntaapi/server/index/IndexableService.java

```diff
@@ -42,81 +89,213 @@ public String getType() {
   public Long getOrderIndex() {
     return orderIndex;
   }
-  
+
   public void setOrderIndex(Long orderIndex) {
     this.orderIndex = orderIndex;
   }
-  
+
   public String getServiceId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 79. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)